### ISSUE TITLE
Training issues，labes.jpg Only the upper left corner has data, and the remaining three have no data

### ISSUE THREAD

**Issue by xiaolieat** [ultralytics/assets#92]
Hello, when I was training with YOLO, I used the following line of command for training, my dataset is a YOLO format dataset downloaded from kaggle, I don't know why after training, there is only the number of categories in the labels.jpg, and there is no data for the other three graphs

yolo train data=YOLO_format/data.yaml model=ultralytics/cfg/models/v8/yolov8n.yaml pretrained=yolov8n.pt epochs=100 batch=64 resume=True

**Comment by UltralyticsAssistant** [comment]
👋 Hello @xiaolieat, thank you for submitting a `ultralytics/assets` 🚀 Issue. To help us address your concern efficiently, please ensure you've provided the following information:

1. For bug reports:
   - A clear and concise description of the bug (you've already done this, thank you!)
   - A minimum reproducible example [MRE](https://docs.ultralytics.com/help/minimum-reproducible-example/) that demonstrates the issue. This is critical to help us diagnose what might be going wrong. In your case, please share:
     - A snippet of your dataset `data.yaml` file
     - Any modifications made to your training command or configuration
     - Any additional logs or outputs related to the issue
   - Your complete environment details:
     - OS and Python version
     - YOLOv8 version and other package versions
   - Expected behavior vs. actual behavior: What did you expect to see in the `labels.jpg`, and what are you observing instead?
   - Hyperparameters or customizations applied during training

2. For questions:
   - Clarify any specific aspects of your workflow you’re unsure about.
   - Provide additional context about your training setup, if relevant.

3. For feature requests:
   - A description of any tool or feature you believe might help address issues like yours in the future.

Additionally, please ensure you've:
- Reviewed the [Ultralytics YOLO Docs](https://docs.ultralytics.com/) for possible solutions
- Searched existing issues in this repository to avoid duplicates

This is an automated response to help get things started, but don't worry—an Ultralytics engineer will also assist you soon. 🚀 If you need to provide more information, please feel free to comment here!

Thank you for your contribution to improving our project! 😊

**Comment by xiaolieat** [comment]
![Image](https://github.com/user-attachments/assets/95747f3b-b573-4d12-9f57-c18416ebdc59)

![Image](https://github.com/user-attachments/assets/549be044-752c-445b-bb15-f0de9bcc3921)

![Image](https://github.com/user-attachments/assets/10f8168f-e818-4acc-9e4a-fa695b6e12a0)

**Comment by xiaolieat** [comment]
> 这是我的标签中的一张图片。jpg ![Image](https://github.com/user-attachments/assets/95747f3b-b573-4d12-9f57-c18416ebdc59)
> 
> 这是我的 YOLO 版本 ![Image](https://github.com/user-attachments/assets/549be044-752c-445b-bb15-f0de9bcc3921)
> 
> 这是我的数据集的配置文件 ![Image](https://github.com/user-attachments/assets/10f8168f-e818-4acc-9e4a-fa695b6e12a0)

**Comment by pderrenger** [comment]
@xiaolieat to address the missing data in your labels.jpg plots, please:

1. **Update** to the latest Ultralytics package: `pip install -U ultralytics`
2. **Verify** your dataset structure matches the required YOLO format shown in the [Ultralytics YOLO Dataset Format documentation](https://docs.ultralytics.com/datasets/detect/)
3. **Check** your label files contain normalized coordinates in `class x_center y_center width height` format

The empty plots typically indicate missing or improperly formatted bounding box annotations in your labels. For assistance creating a test case, see the [Minimum Reproducible Example Guide](https://docs.ultralytics.com/help/minimum-reproducible-example/).
